polychrom/simulation.py

```diff
@@ -743,7 +745,7 @@ def do_block(
                 sys.stdout.flush()
             self._apply_forces()
             self.forces_applied = True
-
+        
```

Comment:
can you apply "black" to the codebase? It will get rid of stray spaces etc.

Reply:
did so with default settings, including on files not part of this PR